docs/api-reference/decorators.md

```diff
@@ -137,6 +137,41 @@ DBOS supports declaration of the following values for `IsolationLevel`:
 
 The transaction semantics of these levels are defined for PostgreSQL [here](https://www.postgresql.org/docs/current/transaction-iso.html).
 
+#### `@StoredProcedure`
+Registers a function as a [DBOS stored procedure](../tutorials/stored-proc-tutorial.md).
+
+The first argument of the decorated function must be a [`StoredProcedureContext`](contexts#storedprocedurecontext), which provides access to the database.
+
+```typescript
+@StoredProcedure({readOnly: true})
+static async doLogin(ctx: StoredProcedureContext, username: string, ) {
+  ...
+}
+```
+
+`@StoredProcedure()` takes an optional `StoredProcedureConfig` object:
+
+```typescript
+interface StoredProcedureConfig {
+  isolationLevel?: IsolationLevel;
+  readOnly?: boolean;
+  executeLocally?: boolean
+} 
+```
+
+The `readOnly` and `isolationLevel` config fields behave the same as their [@Transaction config](contexts#transactioncontextt) counterparts.
+
+The `executeLocally` config field can be used to control where the logic of a stored procedure is executed.
+When `executeLocally` is true, the stored procedure function is executed locally, similar to transaction functions.
```

Comment:
Maybe also explain that this config is mostly useful for debugging purposes. It's unclear why we need this config in the doc.

Reply:
Uhh, these two comments seem a bit contradictory. You want more description on `executeLocally` but you're not clear why we're documenting it?